contrib/renew-root-cert

```diff
@@ -0,0 +1,49 @@
+#!bin/sh -e
+
+# This is an OnChanged handler that can handle requests to renew
+# the factory root CA certificate used to verify the device gateway TLS certificate.
+
+[ -z "$CONFIG_FILE" ] && (echo "No CONFIG_FILE specified"; exit 1)
+[ -f "$CONFIG_FILE" ] || (echo "$CONFIG_FILE does not exist"; exit 1)
+[ -z "$SOTA_DIR" ] && (echo "No SOTA_DIR specified"; exit 1)
+
+fioconfig="$FIOCONFIG_BIN"
+if [ -z "$fioconfig" ] ; then
+	fioconfig=$(which fioconfig)
+fi
+
+correlation_id=$(cat $CONFIG_FILE | grep CORRELATIONID | cut -d= -f2)
+[ -z "$correlation_id" ] && (echo "No CORRELATIONID found in config file"; exit 1)
+echo "Correlation ID for root CA update is $correlation_id"
+
+est_server=$(cat $CONFIG_FILE | grep ESTSERVER | cut -d= -f2)
+[ -z "$est_server" ] && (echo "No ESTSERVER found in config file"; exit 1)
+echo "EST server for update is $est_server"
+
+update() {
+	if ! $fioconfig renew-root $est_server $correlation_id ; then
+		exit 123
+	fi
+	exit 0
+}
+
+# We'll execute under two conditions:
```

Comment:
Are those conditions prevent from updating root CA while there is ongoing root CA update?

Reply:
These conditions were copied from `rotate-cert`, and IIUC they are here to prevent an accidental CA re-renewal unrelated config changes.

Maybe @doanac has a better explaination, as an original author of this (borrowed) idea.